front/lib/api/assistant/global_agents/configurations/dust/shared.ts

```diff
@@ -0,0 +1,98 @@
+import type { MCPServerConfigurationType } from "@app/lib/actions/mcp";
```

Comment:
Wonder if we could actually share the tools prompts. At least data warehouses seems to be the same right ?